x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/steps/create_service_token.tsx

```diff
@@ -58,79 +37,11 @@ export const getGenerateServiceTokenStep = ({
     }),
     status: disabled ? 'disabled' : undefined,
     children: !disabled && (
-      <ServiceTokenStepContent
+      <GenerateServiceTokenComponent
         serviceToken={serviceToken}
         generateServiceToken={generateServiceToken}
         isLoadingServiceToken={isLoadingServiceToken}
       />
     ),
   };
 };
-
-const ServiceTokenStepContent: React.FunctionComponent<{
```

Comment:
moved out and renamed this component to `GenerateServiceTokenComponent` to reuse in generating remote token

Reply:
At the end I haven't used this component somewhere else, but I think the refactor can be left in.